### PR TITLE
(BOLT-274) Strip init name from bolt and task show

### PR DESCRIPTION
**Pull request by adreyer**

### CODE REVIEW

**Comment by MikaelSmith** [comment]
Supposed to be BOLT-274?

File: lib/bolt/cli.rb [hunk 1]
```diff
@@ -526,10 +526,19 @@ def in_bolt_compiler(opts = [])
       end
     end
 
+    # TODO: remove this when the PAL functions return the correct names
+    def fix_typed_name(typed_name)
+      name_parts = typed_name.name_parts
+      if name_parts[-1] == 'init'
```
Comment (MikaelSmith) [inline lib/bolt/cli.rb]: What if the module is named 'init'?

Comment (MikaelSmith) [reply lib/bolt/cli.rb]: I guess for now, we'll still get `init::init` and strip only the last one. So that's probably fine.

Comment (adreyer) [reply lib/bolt/cli.rb]: I would not be surprised if there were bugs elsewhere though.

**Pull request closed without merging**